services/manifesto_model/classifier.py

```diff
@@ -110,7 +110,7 @@ def train(self, data, labels):
             parameters = {'clf__alpha': (10. ** sp.arange(-6, -4, 1.)).tolist()}
 
         # perform gridsearch to get the best regularizer
-        gs_clf = GridSearchCV(text_clf, parameters, n_jobs=-1, verbose=4)
+        gs_clf = GridSearchCV(text_clf, parameters, n_jobs=1, verbose=4)
```

Comment:
warum nicht alle cores?

Reply:
ich glaub es is alles fertig, also crashed zumindest nich mehr und ich hab alles gefixed, was zu fixen war, hoffe ich. 

praesentation is auch erstmal fertig. 

gibt also eigentlich keinen konkreten handlungsbedarf. und heute eh nich fuer dich.

Reply:
ah sorry, falscher thread grade. der scheduler is oefter gecrashed, wegen threading zeugs, da dachte ich, mach ich mal das parallelisieren bei allem aus.

Reply:
ok, gucke ich mir die tage vor der konferenz nochmal an, danke!